unshallow.sh

```diff
@@ -17,8 +17,8 @@ git fetch github --unshallow
 echo "unshallow client"
 cd ~/work/swg/svp/hikey-n-4.9/external/optee_client
 git fetch github --unshallow
-echo "unshallow hello world"
-cd ~/work/swg/svp/hikey-n-4.9/external/optee_hello_world
+echo "unshallow optee_examples"
```

Comment:
I've never really seen this "unshallow.sh" script before. To me it looks like a helper script for some users. It even has hardcoded paths. But you changes here should be fine of course.

Reply:
@jbech-linaro Yes sorry this is for my own use when repo syncing with depth 1 and unshallowing the repos we usually touch before patches can be applied. I can send in a PR later to make the patch relative to pwd.